src/pages/AddLiquidity/index.tsx

```diff
@@ -549,23 +588,27 @@ export default function AddLiquidity({
           hash={txHash}
           content={() => (
             <ConfirmationModalContent
-              title={'Add Liquidity'}
+              title={mustCreateSeparately ? 'Create Pool' : 'Add Liquidity'}
```

Comment:
whoops, I must have missed those when translation the app. created #2030 for tracking